src/praisonai/tests/unit/test_sequential_tool_calling.py

```
@@ -0,0 +1,679 @@
+        mock_completion.side_effect = responses
+        
+        # Create agent
+        agent = Agent(
+            instructions="You are a helpful assistant.",
+            llm='gpt-3.5-turbo',
+            tools=[failing_tool]
+        )
+        
+        # Execute - should handle the error gracefully
+        with pytest.raises(ValueError):
+            result = agent.start("Use the failing tool with value 42.")
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)